### PR TITLE
[Quantum RPG] Fix a few bugs and issues

### PR DESCRIPTION
**Pull request by dstrain115**
Fix a few bugs and issues found when running the RPG game.

- Slime parameter is mainly used for deterministic unit tests.
However, if not specified, it would generate a KeyError.
Change to get() to avoid this exception.
- main loop should pass game state to award XP and not party.
- Tweak a few descriptions that had typos or looked funny.

### CODE REVIEW

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 1]
```diff
@@ -56,14 +56,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
             TALK,
             ["richard", "feynmann", "physicist"],
             (
-                "Richard excitedly talks, as if continuing a lecture in progress.\n"
-                "Modern computers are truly powerful and wondrous, but if want to\n"
+                "Richard excitedly talks, as if continuing a lecture in progress:\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:59]: excitedly talks
speaks excitedly

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 2]
```diff
@@ -56,14 +56,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
+                "'Modern computers are truly powerful and wondrous, but if want to\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:60]: if want
INSERT we

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 3]
```diff
@@ -56,14 +56,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
                 "truly understand nature and the laws of the world around, we are\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:61]: around
APPEND us

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 4]
```diff
@@ -56,14 +56,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
                 "going to need a much more powerful device.\n"
                 "Nature isn't classical, dammit, and if you want to make a simulation\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:63]: dammit
damnit

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 5]
```diff
@@ -141,14 +141,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
-                "David turns to you.  'Welcome to Oxtail University.'\n"
+                "David turns to you.  'Welcome to Oxtail University.' David says.\n"
                 "'You seems to have come at an interesting time.  Our university was\n"
                 "founded here at the forefront of the quantum realm, and we have been\n"
                 "researching quantum computation in search of a computational thoery of\n"
                 "everything.  Unfortunately, our university has been less of an invariant\n"
                 "than hoped.  Quantum fluctuations and errors have infiltrated this whole\n"
                 "place, and we are in dire straits I fear.  Look for my student Artur.\n"
-                "If he still survives, he will be in the communications outpust.\n"
+                "If he still survives, he will be in the communications outpust.i'\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:151]: 1--
he still survives
he's still alive
OR he survived

2--
outpust.i
outpost

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 6]
```diff
@@ -141,14 +141,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
             TALK,
             ["david", "man", "deutsch"],
             (
-                "David turns to you.  'Welcome to Oxtail University.'\n"
+                "David turns to you.  'Welcome to Oxtail University.' David says.\n"
                 "'You seems to have come at an interesting time.  Our university was\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:145]: seems
seem

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 7]
```diff
@@ -141,14 +141,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
                 "founded here at the forefront of the quantum realm, and we have been\n"
                 "researching quantum computation in search of a computational thoery of\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:147]: thoery
theory

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 8]
```diff
@@ -141,14 +141,14 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
                 "everything.  Unfortunately, our university has been less of an invariant\n"
                 "than hoped.  Quantum fluctuations and errors have infiltrated this whole\n"
                 "place, and we are in dire straits I fear.  Look for my student Artur.\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:150]: straits I
INSERT ,

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 9]
```diff
@@ -455,7 +455,7 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
         label="quad3",
         title="Lone Monument",
         description=(
-            "A statue stands in this corner of campus dedicated to Stephen Hawking,"
+            "A statue stands in this corner of campus dedicated to Stephen Hawking,\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:458]: Is the statue or the campus dedicated to Hawking? If it's the statue, this should read:
A statue dedicated to Stephen Hawking stands in this corner of the campus,

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 10]
```diff
@@ -455,7 +455,7 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
             "one of the pioneers of theoretical physics.  A pool of quantum foam\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:459]: PREVIOUS SENTENCE FULL STOP
PREPEND He was

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py [hunk 11]
```diff
@@ -455,7 +455,7 @@ def _green_foam(number: int, prob: float = 0.5, xp=_GREEN_XP):
...
             "surrounds the statue, and nearby, an unnerving tenebrous spot reminiscent\n"
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:460]: statue, and nearby,
statue and, nearby,

**Review by cognigami — CHANGES_REQUESTED** [review]

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 1 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:59]: Done

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 2 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:60]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 3 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:61]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 4 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:63]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 6 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:145]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 7 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:147]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 8 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:150]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 5 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:151]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 10 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:459]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 9 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:458]: Done.

File: unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py (hunk 11 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/classical_realm.py:460]: Done.

**Review by dstrain115 — COMMENTED** [review]
Thanks for catching all my grammar mistakes!

**Review by cognigami — APPROVED** [review]

**Pull request merged**